src/components/SigningOfficialDaaAgreementWrapper.js

```diff
@@ -60,23 +68,27 @@ export const SigningOfficialDaaAgreementWrapper = (props) => {
     }, [
 
       h2({}, [
-        'Agree to Library Card Terms',
+        'Agree to ' + replacementText + ' Terms'
       ]),
 
       p({
         style: {marginBottom: '20px'}
       }, [
-        'To begin issuing Library Cards to researchers from your institution, please review the terms of both data access agreements below and click \'I agree\' when finished.'
+        'To begin issuing ' + replacementText + 's to researchers from your institution, please review the terms of the data access agreement(s) below and click \'I agree\' when finished.'
```

Comment:
I feel like "to begin issuing Data Submitter to researchers..." sounds strange, even though that's the recommendation.

Reply:
I think "to begin issuing Data Submitter privileges to researches" sounds better. What do you think?

Reply:
As a side note, language choices are often required for various reasons. I think in this case, we can make some common-sense changes. In other places, especially around data use questions and phrasing, we need to defer to Jonathan since they often have roots in GA4GH or Broad legal requirements.